pkgs/applications/networking/feedreaders/russ/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, fetchFromGitHub
+, fetchCrate
+, rustPlatform
+, xorg
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "russ";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "ckampfe";
+    repo = "russ";
+    rev = "1482bb1df13738fdd4ea1badf2146a9ed8e6656e";
+    hash = "sha256-MvTMo2q/cQ/LQNdUV8SmHgGlA42kLl0i9mdcoAFV/I4=";
+  };
+
+  cargoHash = "sha256-ObWrwXMGXkLqqM7VXhOXArshk2lVkbOTXhrQImDQp1s=";
+
+  nativeBuildInputs = [ xorg.libxcb ];
+
+  # tests are network based :(
+  doCheck = false;
+
+  meta = with lib; {
```

Comment:
I see russ maintains CHANGELOG, can you please update `meta` to include url to changelog.

https://nixos.org/manual/nixpkgs/stable/#var-meta-changelog